qiskit/circuit/library/__init__.py

```diff
@@ -501,6 +501,7 @@
     Permutation,
     PermutationGate,
     GMS,
+    MCMTGate,
```

Comment:
You should also add it to line 188 in this file - it currently does not appear in the circuit library docs!